converter/download.py

```diff
@@ -6,7 +6,7 @@
 def download_result():
     try:
         return send_file(current_app.config['OUTPUT_FILE'], 
-            attachment_filename="result.abc" ,as_attachment=True)
+            attachment_filename="result.abc" ,as_attachment=True, cache_timeout=0)
```

Comment:
Why is cache_timeout is set 0 here? What is the default value?

Reply:
if cache_timeout is not set to 0, the server will not send updated result file when I test download locally